src/clients/ksu/ccache.c

```diff
@@ -64,12 +65,13 @@ krb5_ccache_copy(krb5_context context, krb5_ccache cc_def,
                  krb5_boolean restrict_creds, krb5_principal primary_principal,
                  krb5_boolean *stored)
 {
-    krb5_error_code retval;
+    krb5_error_code retval=0;
```

Comment:
I am not a fan of setting the return code to "success" by default, especially since we use `goto`. But I guess the fact we reuse `retval` for each sub-call would make the security benefit of the "error by default" approach limited.

Reply:
Unless I made a mistake here, I was just reverting a change that wasn't necessary for the purpose of the PR.  In general I don't initialize `ret` type variables in new code, but if we start doing general cleanup to the ksu source code we will swamp the memory leak fixes.